commit 0d2a449dce07ead193ae6f41e07a2b6ca0e0728b (refs/changes/88/388/2)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-30T23:26:37+00:00 (3 years, 1 month ago)
    
    MANTA-2977 move sdc-manta to cr.joyent.us
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/README.md b/README.md
index 087b00f..0b81d28 100644
--- a/README.md
+++ b/README.md
@@ -5,14 +5,15 @@
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright (c) 2016, Joyent, Inc.
 -->
 
 # sdc-manta
 
-This repository is part of the Joyent SmartDataCenter project (SDC).  For
-contribution guidelines, issues, and general documentation, visit the main
-[SDC](http://github.com/joyent/sdc) project page.
+This repository is part of the Joyent Triton project. See the [contribution
+guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
+*Triton does not use GitHub PRs* -- and general documentation at the main
+[Triton project](https://github.com/joyent/triton) page.
 
 This repository contains all the deployment tools necessary to manage a Manta
 within an SDC deployment.
